src/components/organisms/PaneManager/PaneManagerSplitView.tsx

```diff
@@ -25,139 +20,61 @@ const TemplateManagerPane = React.lazy(() => import('@organisms/TemplateManagerP
-              maxConstraints={[leftPaneMaxWidth, paneHeight]}
-              axis="x"
-              resizeHandles={['e']}
-              handle={(h: number, ref: LegacyRef<HTMLSpanElement>) => (
-                <span className="custom-modal-handle" ref={ref} />
-              )}
-              onResizeStop={resizeLeftPane}
-            >
-              <S.Pane id="LeftPane">
-                <Suspense fallback={null}>
-                  {leftMenuSelection === 'file-explorer' && <FileTreePane />}
-                  {leftMenuSelection === 'helm-pane' && <HelmPane />}
-                  {leftMenuSelection === 'kustomize-pane' && <KustomizePane />}
-                  {leftMenuSelection === 'templates-pane' && <TemplateManagerPane contentHeight={paneHeight} />}
-                </Suspense>
-              </S.Pane>
-            </ResizableBox>
-          </S.LeftPaneContainer>
+          <ReflexElement id="leftPane" minSize={MIN_SPLIT_VIEW_PANE_WIDTH} flex={layout.leftPane}>
+            <S.LeftPane>
+              <Suspense fallback={<div />}>
+                {leftMenuSelection === 'file-explorer' && <FileTreePane />}
+                {leftMenuSelection === 'helm-pane' && <HelmPane />}
+                {leftMenuSelection === 'kustomize-pane' && <KustomizePane />}
+                {leftMenuSelection === 'templates-pane' && <TemplateManagerPane contentHeight={paneHeight} />}
+              </Suspense>
+            </S.LeftPane>
+          </ReflexElement>
         )}
 
-        <S.Pane id="NavPane" $height={paneHeight} ref={navPaneRef}>
+        {leftActive && leftMenuSelection && <ReflexSplitter propagate />}
+
+        <ReflexElement
+          id="navPane"
+          minSize={MIN_SPLIT_VIEW_PANE_WIDTH}
+          maxSize={MIN_SPLIT_VIEW_PANE_WIDTH + 75}
+          flex={layout.navPane}
+        >
           <NavigatorPane />
-        </S.Pane>
-
-        <S.EditorPaneContainer ref={editPaneRef}>
-          <ResizableBox
-            height={paneHeight}
-            width={splitViewContainerWidth * editWidth}
-            minConstraints={[MIN_SPLIT_VIEW_PANE_WIDTH, paneHeight]}
-            maxConstraints={[editPaneMaxWidth, paneHeight]}
-            axis="x"
-            resizeHandles={['w']}
-            handle={(h: number, ref: LegacyRef<HTMLSpanElement>) => <span className="custom-modal-handle" ref={ref} />}
-            onResizeStop={resizeEditPane}
-          >
-            <S.Pane id="EditorPane" $height={paneHeight}>
-              <ActionsPane contentHeight={paneHeight} />
-            </S.Pane>
-          </ResizableBox>
-        </S.EditorPaneContainer>
-
-        {featureJson.ShowGraphView && rightMenuSelection === 'graph' && rightActive && (
-          <S.Pane id="RightPane" $height={paneHeight}>
-            <GraphView editorHeight={paneHeight} />
-          </S.Pane>
-        )}
-      </S.SplitViewGrid>
+        </ReflexElement>
+
+        <ReflexSplitter propagate={leftActive && leftMenuSelection !== undefined} />
```

Comment:
📓 just to spread some awareness: you can only use propagate when you have multiple splitters or it gives unexpected behavior.

Reply:
I would add this as a comment as well